### PR TITLE
Meeting notes 2020 October 26th

### PR DESCRIPTION
**Pull request by cmarmo**

### CODE REVIEW

File: meeting_notes/2020-10-26.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+### General topics
+- [0.24 Release](https://github.com/scikit-learn/scikit-learn/pulls?q=is%3Aopen+is%3Apr+milestone%3A0.24):
+  - pair releasing: Olivier and Chiara
+  - Chiara will review the [milestone](https://github.com/scikit-learn/scikit-learn/pulls?q=is%3Aopen+is%3Apr+milestone%3A0.24) list, remove items if necessary,
+    ping reviewers or contributors if necessary
+  - pull requests from the milestone will be pushed during this week, next Monday Olivier will create the release branch
+  - Olivier: HGBT and categorical features is still challenging; we don't need to block the PR and can handle the ease of use in future PRs.
+  - Check CircleCI checks: nice to fix all the failures for the release.
+  -  documentation and nightly builds are no longer hosted on rackspace, but at anaconda.org ([#18652](https://github.com/scikit-learn/scikit-learn/pull/18652))
+- 1.0 Release: to be kept in mind when selecting or splitting PRs currently in 0.24 milestone.
```
Comment (NicolasHug) [inline meeting_notes/2020-10-26.md:12]: I'm not sure I understand this point

**Review by NicolasHug — APPROVED** [review]
thanks @cmarmo

File: meeting_notes/2020-10-26.md (hunk 1 above)
Comment (cmarmo) [reply meeting_notes/2020-10-26.md:12]: In my understanding, some complicated PRs are often split in small parts to make review and merging easier and faster.
This is ok when you move forward using a small steps approach: having 1.0 behind the corner, gives you a different perspective. Perhaps some problems could be solved if a PR is just abandoned and the problem completely reformulated: 1.0 is a good place to do that.
If this is not clear, I would be happy to remove... I'm not enough into scikit-learn API to give you examples.

Comment (NicolasHug) [reply meeting_notes/2020-10-26.md:12]: Thanks for the details.

We decided in the previous meetings that 1.0 would just be a very normal release without backward incompatibility, so I'm not sure we can really change our incremental approach here (IIUC)

Comment (cmarmo) [reply meeting_notes/2020-10-26.md:12]: > We decided in the previous meetings that 1.0 would just be a very normal release without backward incompatibility

Well, definitely I'm having some hard time in following what decisions are taken during the meetings ;).
Anyway, I will remove this line, if this make people happier.

Comment (NicolasHug) [reply meeting_notes/2020-10-26.md]: you're not alone ;) that one was particularly confusing to me

some follow up here, for ref: https://github.com/scikit-learn/scikit-learn/issues/14386#issuecomment-683890328

**Review by lorentzenchr — APPROVED** [review]
LGTM

**Pull request merged**